src/main/java/stsjorbsmod/cards/cull/CULLCard.java

```diff
@@ -41,7 +44,7 @@ public CULLCard() {
 
     @Override
     public void use(AbstractPlayer p, AbstractMonster m) {
-        this.addToBot(new RitualDaggerAction(m, new DamageInfo(p, damage, damageTypeForTurn), magicNumber, uuid));
+        this.addToBot(new DamageAction(m, new DamageInfo(p, damage, damageTypeForTurn), AbstractGameAction.AttackEffect.SLASH_VERTICAL));
```

Comment:
Why this change? Isn't it desirable for it to upgrade even if the first swing kills?

Reply:
Yeah, forgot about that. Though, with `RitualDaggerAction` done twice, if the fight doesn't end, Cull card damage increases twice. Working on a new Action for it.

Reply:
We'd probably need a new action anyway since CULL is also supposed to upgrade on minion kills

Reply:
oh right.